docs/changelog/85551.yaml

```
@@ -0,0 +1,5 @@
+pr: 85534
+summary: "Fix NPE in RepositoriesService"
```

Think this would be better for the release notes:

```suggestion
summary: "Distinguish missing and invalid repositories"
```